frontend/src/entities/configuration/configuration-reducer.ts

```diff
@@ -78,15 +88,44 @@ export const AppConfigSlice = createSlice({
                 const { data } = action.payload;
                 return {
                     ...state,
-                    appConfig: data[0],
                     appConfigList: data,
                     failedToLoadConfig: false,
+                    loadingAppConfig: false,
+                };
+                
+            })
+            .addMatcher(isPending(listConfigurations), (state) => {
+                return {
+                    ...state,
+                    loadingAppConfig: true,
                 };
             })
             .addMatcher(isRejected(listConfigurations), (state) => {
                 return {
                     ...state,
                     failedToLoadConfig: true,
+                    loadingAppConfig: false,
+                };
+            })
+            .addMatcher(isFulfilled(getConfiguration), (state, action: any) => {
+                const { data } = action.payload;
+                return {
+                    ...state,
+                    appConfig: data[0],
+                    loadingAppConfig: false,
```

Comment:
Should `loadingAppConfig` be used for `getConfiguration` too? It looks like we use it for both get and list, so if one starts loading this will be true, and if one finishes loading before the other then this will be false for both. 

It looks like you're specifically using this for list, so perhaps just change it to `loadingAppConfigList` and only use it for the list call.